en/sim_gazebo_gz/gazebo-models.md

```diff
@@ -0,0 +1,73 @@
+# Gazebo Models Repository (PX4-gazebo-models)
+
+The [PX4-gazebo-models](https://github.com/PX4/PX4-gazebo-models) repository is used to store all [Gazebo](../sim_gazebo_gz/README.md) models and worlds that are supported by PX4.
+
+- Models are stored in the `/models` directory and worlds are stored in the `/worlds` directory.
+- The [simulation-gazebo](https://github.com/PX4/PX4-gazebo-models/blob/main/simulation-gazebo) Python script is used for starting Gazebo in standalone mode.
+  This script can communicate with a PX4 SITL instance on the same host by default.
+  If the script parameters are set correctly, it can also communicate with any PX4 instance on any machine within the same network.
+
+## Simulation Start-up
+
+The `simulation-gazebo` script does not require any additional libraries and should work out of the box.
+The default `simulation-script` can be run with:
+
+```sh
+python simulation-gazebo
+```
+
+With these default parameters, there will first be a check whether a folder called `/.simulation-gazebo` already exists.
+If if does not, all models and worlds will be downloaded from [PX4 gazebo models repository](https://github.com/PX4/PX4-gazebo-models) and placed there.
```

Comment:
Since I don't do anything specific to fetch or call this normally,  presumably PX4 calls this simulation-gazebo at least once?

Reply:
Not the script itself. But it goes through some of the same motions. It checks for the existence of a folder called `/.simulation-gazebo` and downloads it from github if it doesn't exist or the `PX4_GZ_OVERWRITE` parameter has been set. It is set here: https://github.com/PX4/PX4-Autopilot/blob/d8807ac894ff7f8c139c9f1691c2a713f84a0029/ROMFS/px4fmu_common/init.d-posix/px4-rc.simulator#L75

Note that you don't get the same kind of freedoms when you run it this way (no custom storage directory option, no custom download directory option). Plus you also don't get to set the GZ env variables as mentioned above, so you cannot run PX4 and GZ separately.